# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webcodecs/moz.build
# Commit: bc4f18f14bd3
# Full Hash: bc4f18f14bd35e1d98ec9164447fa10ac0bfc332
# Author: Paul Adenot <paul@paul.cx>
# Date: 2023-12-23 09:52:03
# Regressor Bug: 1749047
# File Overlap Count: 1
# Description:
#   Bug 1749047 - Implement the boilerplate for VideoEncoder. r=chunmin,webidl,smaug,saschanaz
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D195705
# ==============================================================================

diff -r ee31e639b046 -r bc4f18f14bd3 dom/media/webcodecs/moz.build
--- a/dom/media/webcodecs/moz.build	Fri Dec 22 21:39:06 2023 +0000
+++ b/dom/media/webcodecs/moz.build	Fri Dec 22 21:39:07 2023 +0000
@@ -27,6 +27,7 @@
     "EncoderTypes.h",
     "VideoColorSpace.h",
     "VideoDecoder.h",
+    "VideoEncoder.h",
     "VideoFrame.h",
     "WebCodecsUtils.h",
 ]
@@ -37,6 +38,7 @@
     "EncodedVideoChunk.cpp",
     "VideoColorSpace.cpp",
     "VideoDecoder.cpp",
+    "VideoEncoder.cpp",
     "VideoFrame.cpp",
     "WebCodecsUtils.cpp",
 ]